### DISCUSSION TITLE
Naming convention for skeleton.py: rename or leave as-is?

### DISCUSSION THREAD

**Discussion by sgbaird** [pyscaffold/pyscaffold#638] [Best Practices & Workflows]
It's not entirely clear to me: is the intention for the user to rename `skeleton.py` to something more consistent with the project or leave it named `skeleton.py`? I would assume the former followed by updating any references to the skeleton module being renamed as well.

For example, I took after some of the structure of [UMAP's repo](https://github.com/lmcinnes/umap/tree/master/umap) in several of mine, where if the package name / directory is called `mat_discover` then I make a module called `mat_discover_.py`:
<img src=https://user-images.githubusercontent.com/45469701/169591804-bf37c7ec-f6d2-4dbc-868a-a08f695c138d.png width=300>

Alternatively I could call it `core.py` for example.

**Comment by FlorianWilhelm** [comment]
Hi @sgbaird, the `skeleton.py` file can be safely removed as stated in the docstring but could also be renamed to whatever you want depending on your needs. I typically rename it to `cli.py` for command-line-interface but I am not sure this really is a convention.

Does this answer your question? Do you have a suggestion what we could add to the docstring so that this becomes clearer?

**Reply by sgbaird** [reply]
It was pretty clear to me that `skeleton.py` could be safely removed. Yes - definitely answers my question. Perhaps "`skeleton.py` can be renamed as desired. For example, `cli.py`, `core.py`, or `<my_module_name>.py` (replacing `<my_module_name>` with the name of your module)." or I think your wording would work just fine:
"`skeleton.py` can be left as-is or renamed to whatever you want depending on your needs."

Thank you for clarifying!

**Reply by FlorianWilhelm** [reply]
Cool, how about:

```
Note:
    This skeleton file can be renamed depending on your needs or safely removed if not needed.
```
If you want to provide a PR we appreciate it 🙂 Otherwise, I'll change this after my vacation.

**Reply by abravalheri** [reply]
I was thinking about cutting a release soon, so I have created #641.
